app/views/file_manager/index.html.erb

```diff
@@ -0,0 +1,124 @@
+<% content_for :javascripts do %>
+  <script>
+      function rename(path) {
+          var new_name = prompt("Enter the new name:");
+          var rel_path = path.split("/file_manager/")[1].replace(/%40/g, '@');
+          $.ajax({
+              url: "/file_manager/" + rel_path,
+              type: "PUT",
+              data: { new_name: new_name, relative_path: rel_path},
+              success: function(data) {
+                  console.log(`Renamed: ${rel_path}`)
+                  location.reload();
+              }
+          });
+      }
+
+      function deleteSelected(path) {
+          $.ajax({
+              url: path,
+              type: "DELETE",
+              headers: {
+                  'X-CSRF-Token': $('meta[name="csrf-token"]').attr('content'),
+              },
+              success: function () {
+                  console.log(`Deleted: ${path}`);
+                  location.reload();
+              },
+              error: function (xhr, status, error) {
+                  console.error(`Failed to delete ${path}: ${error}`);
+              }
+          });
```

Comment:
Note that this isn't needed since `app-level.js` contains the following code:

``` js
	/* Adding CSRF token to every Ajax call by default */
	jQuery.ajaxSetup({
    	headers: {
        	'X-CSRF-Token' : $('meta[name="csrf-token"]').attr('content')
    	},
	});
```